src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -349,21 +349,46 @@ void AuctionHouseMgr::LoadAuctions()
     uint32 oldMSTime = getMSTime();
 
     PreparedStatement* stmt = CharacterDatabase.GetPreparedStatement(CHAR_SEL_AUCTIONS);
-    PreparedQueryResult result = CharacterDatabase.Query(stmt);
+    PreparedQueryResult resultAuctions = CharacterDatabase.Query(stmt);
 
-    if (!result)
+    if (!resultAuctions)
     {
         TC_LOG_INFO("server.loading", ">> Loaded 0 auctions. DB table `auctionhouse` is empty.");
 
         return;
     }
 
-    uint32 count = 0;
+    // parse bidder list
+    std::unordered_map<uint32, std::unordered_set<ObjectGuid::LowType>> bidderlists;
+    PreparedStatement* stmt2 = CharacterDatabase.GetPreparedStatement(CHAR_SEL_AUCTION_BIDDERS);
+    
+    uint32 countBidders = 0;
+    if (PreparedQueryResult resultBidders = CharacterDatabase.Query(stmt2))
+    {
+        do
+        {
+            Field* fields = resultBidders->Fetch();
+            uint32 auctionId = fields[0].GetUInt32();
 
+            auto it = bidderlists.find(auctionId);
```

Comment:
not needed, operator[] creates entries in map if not present.
just bidderlists[fields[0].GetUInt32()].insert(fields[1].GetUInt32());